### PR TITLE
Improve layout of vulnerability subsection.

### PR DESCRIPTION
**Pull request by hogo6002**
Issue: https://github.com/google/osv.dev/issues/797
Before:
image

After:
image

### CODE REVIEW

**Review by another-rex — DISMISSED** [review]
LGTM

File: gcp/appengine/frontend3/src/styles.scss [hunk 1]
```diff
@@ -598,6 +598,7 @@ dl.vulnerability-details,
   grid-template-columns: max-content minmax(0, auto);
   grid-row-gap: 5px;
   gap: 8px;
+  margin-bottom: 15px;
```
Comment (oliverchang) [inline gcp/appengine/frontend3/src/styles.scss:601]: Very nice! 

Super nit: any chance we can use https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child to not add the extra whitespace in the last entry?

Comment (hogo6002) [reply gcp/appengine/frontend3/src/styles.scss]: Done, added a not(last-child) selector for .vulnerability-package-subsection dl.

File: gcp/appengine/frontend3/src/styles.scss [hunk 2]
```diff
@@ -669,6 +669,10 @@ dl.vulnerability-details,
   }
 }
 
+.vulnerability-package-subsection dl:not(:last-child) {
+  margin-bottom: 15px;
```
Comment (another-rex) [inline gcp/appengine/frontend3/src/styles.scss:673]: Can you put this in the nested area you had it in before, and use the & operator to apply the last-child property.

See https://sass-lang.com/documentation/style-rules/parent-selector/

**Review by another-rex — CHANGES_REQUESTED** [review]

File: gcp/appengine/frontend3/src/styles.scss (hunk 2 above)
Comment (hogo6002) [reply gcp/appengine/frontend3/src/styles.scss]: Thanks!

**Review by another-rex — APPROVED** [review]

**Pull request merged**